msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -23,41 +25,49 @@ Teams supports access to apps during meeting for the following meeting types:
+* **Organizer**: The organizer schedules a meeting, sets the meeting options, assigns meeting roles, and starts the meeting. The users with Microsoft 365 account and Teams license can only be the organizers, and control attendee permissions. A meeting organizer can change the settings for a specific meeting. Organizers can make these changes on the **Meeting options** web page.
 
-This article provides an overview of meeting app extensibility, API references, enable and configure apps for meetings, and custom Together Mode scenes in Teams.
+* **Presenter**: The presenters have same capabilities of the organizers with exclusions. A presenter can't remove an organizer from the session or modify meeting options for the session.
 
-* **Extend meeting app**: Enhance your meeting experience by using the meeting extensibility feature. This feature enables you to integrate your apps within meetings. It also includes different stages of a meeting lifecycle, where you can integrate tabs, bots, and message extensions. You can identify various participant roles and user types, get meeting events, and generate in-meeting dialogs.
-* **Configure apps for meetings**: To customize Teams with apps for meetings, enable your apps for Teams meetings by updating the app manifest and also configure the apps for meeting scenarios.
-* **Customize with Together Mode scenes**: The new custom Together Mode scenes feature enables users to collaborate in a meeting with their team in one place.
-* **Customize app permission in shared channel**: If your app shares important information in shared channel, you can customize the app permission for external members. App permissions in [Shared channels](../concepts/build-and-test/Shared-channels.md) follow host team's app roster and host tenant's app policy.
-* **Retrieve meeting transcripts**: You can access and retrieve meeting transcripts in a post-meeting scenario. Configure your app to get transcripts automatically for a scheduled meeting, and use them for insights, intelligent analysis, and more.
+* **Attendee**: An attendee is a user who is invited to attend the meeting. Attendees have limited capabilities during the meeting.
 
-## Next step
+> [!NOTE]
+> Only an organizer or presenter can add, remove, or uninstall apps.
 
-> [!div class="nextstepaction"]
-> [Unified meetings apps](meeting-app-extensibility.md)
+For more information, see [roles in a Teams meeting](https://support.microsoft.com/office/roles-in-a-teams-meeting-c16fa7d0-1666-4dde-8686-0a0bfe16e019).
+
+After you design your app based on participant roles in a meeting, you can identify each user type for meetings and select what they can access.
+
+> [!TIP]
+>
+> * The default participant settings are determined by an organization's IT administrator. As per default settings, participants joining a meeting have the presenter role.
+> * Participant role isn't available in one-on-one call.
```

Comment:
Did you mean presenter? Participant isn't one of the roles above.

Reply:
@v-sdhakshina pls update "Presenter" instead of "Participant"